cpg_infra/billing_aggregator/aggregate/utils.py

```diff
@@ -599,35 +624,34 @@ async def _get_jobs_and_add_to_queue(
         """
         batch_id = batch['id']
         jobs_cnt = batch['n_jobs']
-
-        # only get the first jb if there are more than DEFAULT_MAX_JOBS_PER_BATCH jobs
-        limit = 1 if jobs_cnt > DEFAULT_MAX_JOBS_PER_BATCH else None
-        async for jobs in get_jobs_for_batch(batch_id, token, limit):
-            if jobs_cnt <= DEFAULT_MAX_JOBS_PER_BATCH:
+        batch_name = batch.get('attributes', {}).get('name')
+
+        # check if hail batch query or too many jobs
+        if batch_name is None or jobs_cnt > DEFAULT_MAX_JOBS_PER_BATCH:
```

Comment:
- We have batches over the max_jobs_per_batch that we want to process
- We have batches without names under the count we want to process

```suggestion
        if batch_name is None and jobs_cnt > DEFAULT_MAX_JOBS_PER_BATCH:
```

Reply:
Ok, that makes sense, I have thought we want to skip very large ones no matter if HB Query or not. Will change to 'and' so only very large HB Query will be aggregated.